test/types/optionSelect.test.tsx

```diff
@@ -58,4 +61,32 @@ describe('optionSelect', () => {
     expect(tester.text()).not.toContain('Yes');
     expect(tester.text()).not.toContain('2 Values');
   });
+
+  it('Shows search for large data sets, or on command', async () => {
+    // tslint:disable no-magic-numbers
+    const states = Array(50).map((_v: any) => ({
```

Comment:
This `50` should be replaced by something based off `SHOW_SEARCH_IF_OVER` like `SHOW_SEARCH_IF_OVER + 1`